src/Common/Client/Modules/AbstractFeatures.php

```diff
@@ -46,7 +46,8 @@ private function getServerRequestInterface(AbstractRequest $request): ServerRequ
 
         $uri = self::forgeUri($endpointUri, $serverRequestInterface->getUri());
 
-        return $this->addAuthorization($serverRequestInterface->withUri($uri));
+        return $this->addMessageIds($serverRequestInterface, $request)
```

Comment:
Hello,
this fails with an error, because addMessageIds returns a ServerRequestInterface, which does not have an addAuthorization method. I have fixed it in this branch: https://github.com/mrbig/ocpi-protocol/tree/v2.2.1-messageIds

Reply:
Nice catch, thanks!